config/routes.rb

```diff
@@ -42,6 +42,8 @@
 
   put "resources/:id/onesky", to: "resources#push_to_onesky"
 
+  get "/translations/files/:path", to: redirect("https://#{ENV.fetch("MOBILE_CONTENT_API_BUCKET")}.s3.#{ENV.fetch("AWS_REGION")}.amazonaws.com/%{path}", status: 302)
```

Comment:
for organization should we upload files to the s3 bucket under a `files` subdirectory or something similar?

Reply:
in that bucket we currently have:
- an attachments folder
- a GodTools folder (GodTools here references the "system" configured in the mobile-content-api database, there will be additional systems in the future)
- and a bunch of files in the root directory (I believe this is the new place attachments get written by rails)

I think we should put the files in a `/translations/files/` subdirectory within S3 to prevent us from continuting to overload the root directory with all different types of files

Reply:
Sure, I can update the paths for that